static/src/stylesheets/content.scss

```diff
@@ -1,14 +1,13 @@
 @charset 'UTF-8';
 
+@import '_facia-content';
+
 /* ==========================================================================
    Variables and mixins
    ========================================================================== */
 
-@import '_vars';
```

Comment:
i'm not sure about abstracting these, they're kind of 'settings' partials, it's not clear there's present now?